Added Places to server and hopefully fixed duplicate pages bug

alveerak opened the pull request:
Also merged dimens.xml and dimens2.xml, moved all string constants in login activity and page class to constants.java.

samhithak commented:
This looks like a lot of duplicate code. Is there any chance we can refactor this to not have so much of duplicate code? I am okay to merge it to unblock you now, but would highly suggest to refactor this as soon as you can since it would be difficult to maintain this code later. We can discuss about this if you have any questions. 

Looks like you have merge conflicts. I will let you resolve those to merge

alveerak commented:
I opened a new pull request for the refactored code. There were 3 functions that I added to get Hometown, current location, and tagged places. I refactored hometown and current location to be the same function with extra parameters. Tagged places deals with an array and is more different from the other two so I left it as is.
